source/js/classes/crop-host.js

```diff
@@ -379,22 +398,20 @@ crop.factory('cropHost', ['$document', 'cropAreaCircle', 'cropAreaSquare', 'crop
         theArea = new CropAreaCircle(ctx, events);
 
         // Init Mouse Event Listeners
-        $document.on('mousemove', onMouseMove);
+        elCanvas.on('mousemove', onMousehover);
```

Comment:
Why is there a differentiation between mousemove and mousehover?

Also it looks like the mousemove is still only being done when focused inside the canvas? Resizing the crop area has to be possible otuside of the canvas while mousebutton is down